pwndbg/gdblib/regs.py

```diff
@@ -48,21 +37,15 @@ class module(ModuleType):
     def __getattr__(self, attr: str) -> int:
         attr = attr.lstrip("$")
         try:
-            # Seriously, gdb? Only accepts uint32.
-            if "eflags" in attr or "cpsr" in attr:
-                value = gdb77_get_register(attr)
-                value = value.cast(pwndbg.gdblib.typeinfo.uint32)
-            else:
-                value = get_register(attr)
-                if value is None and attr.lower() == "xpsr":
-                    value = get_register("xPSR")
-                size = pwndbg.gdblib.typeinfo.unsigned.get(
-                    value.type.sizeof, pwndbg.gdblib.typeinfo.ulong
-                )
-                value = value.cast(size)
-                if attr.lower() == "pc" and pwndbg.gdblib.arch.current == "i8086":
-                    value += self.cs * 16
-
+            value = gdb_get_register(attr)
+            if value is None and attr.lower() == "xpsr":
+                value = gdb_get_register("xPSR")
+            size = pwndbg.gdblib.typeinfo.unsigned.get(
+                value.type.sizeof, pwndbg.gdblib.typeinfo.ulong
+            )
+            value = value.cast(size)
+            if attr.lower() == "pc" and pwndbg.gdblib.arch.current == "i8086":
```

Comment:
I really wonder if we have to do `.lower()` here at all, I think we should never have here an uppercase value ( ? )

Would need to check different archs if we have there uppercase reg names lol